src/screens/ProfileScreen/DetailScreen/ExtensionRequest.tsx

```diff
@@ -94,7 +91,9 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: 'white',
```

Comment:
use color variable

Reply:
for color variable, we need to add a new issue  as we have to update it everywhere with testing.